tests/clickhouse-test

```diff
@@ -1993,6 +1993,27 @@ class TestSuite:
             else:
                 self.parallel_tests.append(test_name)
 
+        # Sort sequential tests so high-load tests will be run
+        # at the end of the suite when most of parallel tests are finished.
+        # This is helpful when parallel tests and sequential tests run concurrently.
+
+        def tests_comparator(test: str):
+            if test == "00002_log_and_exception_messages_formatting.sql":
```

Comment:
Maybe it will be better to avoid random tests names and instead, use tags, like `no-batch` or introduce one more i.e. `run-last` (there should be a better name)?

Reply:
This test is special, it's whole functionality should be moved into function call at the end of suite.
`no-batch` is already introduced for this test)